src/api/routers/participantsRouter.ts

```diff
@@ -277,6 +277,15 @@ export function createParticipantsRouter() {
     }
   );
 
+  participantsRouter.post(
+    '/:participantId/completedRecommendationsWorkflow',
```

Comment:
I thought about just adding this functionality to the `/:participantId/sharingPermission/add` endpoint and checking if `newTypes` was defined, but that would not actually confirm if the user had gone through the recommendations because we may populate that field for them.